spec/tests/yaml/swiftnav/sbp/test_piksi.yaml

```diff
@@ -0,0 +1,561 @@
+---
+description: Unit tests for swiftnav.sbp.piksi v0.23.
+generated_on: 2015-03-24 11:31:59.676598
+package: sbp.piksi
+tests:
+
+- msg:
+    fields: null
```

Comment:
I agree about having a field to make it human readable, but how about just making the existing print message have a single string field? Should maintain binary compatibility.

I would like to deprecate and move to a new type when we decide what to do about encoding the logging level. Until we make that decision I'm hesitant to make a new message. Unless we think we can decide that right away?